frontend/src/components/AppHeader.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components/macro'
+
+export default function AppHeader() {
+  return (
+    <Header>
+      <h1>musiCompass</h1>
+      <h5>...discover new sounds!</h5>
+    </Header>
+  )
+}
+
+const Header = styled.header`
+  background: var(--primary-color);
+  box-shadow: 0px 2px 8px #333;
+  padding: 10px 0px 20px 0px;
+  h1 {
+    margin: 0px;
+    padding: 8px;
+    text-align: center;
+    font-size: 1.6em;
+    font-family: 'Orange Juice';
```

Comment:
you should be able to put duplicate attributes like font-family into the header component to reduce redundant code

Reply:
thx. changed